src/main/java/org/apache/commons/io/FilenameUtils.java

```diff
@@ -665,7 +666,10 @@ public static int getPrefixLength(final String fileName) {
         if (ch1 == ':') {
             ch0 = Character.toUpperCase(ch0);
             if (ch0 >= 'A' && ch0 <= 'Z') {
-                if (len == 2 || isSeparator(fileName.charAt(2)) == false) {
+                if (len == 2 && SystemUtils.IS_OS_LINUX) {
```

Comment:
I think you can use "!FilenameUtils.isSystemWindows()" instead. see https://github.com/apache/commons-io/blob/master/src/main/java/org/apache/commons/io/FilenameUtils.java#L146

Reply:
thanks, updated to use  "!FilenameUtils.isSystemWindows()"

Reply:
I though about this a little more and since other OSs support driver letters, I've abstracted this away in a new method `FileSystem#supportsDriveLetter()`, so please rebase on master and use that method instead of testing for the OS directly.

Reply:
FileSystem.supportsDriveLetter() is non-static method getPrefixLength is static

Reply:
So?